### PR TITLE
fix broken links in README

### PR DESCRIPTION
**Pull request by neha-viswanathan**
Signed-off-by: Neha Viswanathan <itsnehavis@gmail.com>

**Which issue(s) this PR fixes**:

Fixes #https://github.com/kubernetes/website/issues/23964

### CODE REVIEW

**Comment by neha-viswanathan** [comment]
I signed it

**Review by markyjackson-taulia — APPROVED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -66,9 +66,9 @@ We encourage all contributors to become members. We aim to grow an active, healt
...
 contributors/guide/README.md
 [file an issue]:
-/contributors/guide/README.md#file-an-issue
+contributors/guide/first-contribution.md#file-an-issue
 [find something to work on]:
-contributors/guide/README.md#find-something-to-work-on
+contributors/guide/first-contribution.md#find-something-to-work-on
 [open a pull request]:
-contributors/guide/README.md#open-a-pull-request
+contributors/guide/contributing.md#open-a-pull-request
```
Comment (spiffxp) [inline README.md:73]:
```suggestion
contributors/guide/contributing.md#opening-a-pull-request
```

**Review by cblecker — APPROVED** [review]

**Pull request merged**